### DISCUSSION TITLE
it  return "405 method not allowed" when i  put an "put" request

### DISCUSSION THREAD

**Discussion by chenjianquan7** [DependencyTrack/dependency-track#3004] [General]
it contain the token,
![image](https://github.com/DependencyTrack/dependency-track/assets/26828709/c7dd5c9d-c76e-4212-8844-55212f21998f)

![image](https://github.com/DependencyTrack/dependency-track/assets/26828709/b80930bf-fbcf-4264-a12e-1c60b85d5d76)
![image](https://github.com/DependencyTrack/dependency-track/assets/26828709/a317b708-80db-4b0f-81de-93c21c59b145)

**Comment by syalioune** [comment]
Hello @chenjianquan7 
I'm not sure to understand : which endpoint are you trying to call <Method> <URI> ?

**Reply by chenjianquan7** [reply]
i am create an project use rest api,   
put /v1/project

**Reply by syalioune** [reply]
Thanks for your answer, it seems like you are missing the `api` part in your call : `PUT /api/v1/project`
